0-Risk/Scribbles
# #171110020 Integrate Travis Coveralls code coverage service along with their readme badges

#### What does this PR do?

- Connect Scribbles to Travis account through GitHub for continuous integration testing for every commit

Integrate coverage with Travis

#### Description of Task to be completed?

- Configure coveralls and coverage in the travis build life cycle through travis.yml

#### How should this be manually tested?

- check Scribbles Travis account

- check Scribbles coveralls account

#### What are the relevant pivotal tracker stories?

- [#171110020](https://www.pivotaltracker.com/story/show/171110020)
